DataFormats/Detectors/CTP/include/DataFormatsCTP/Digits.h

```diff
@@ -29,17 +29,21 @@ static constexpr std::uint32_t NumOfHBInTF = 256;
 static constexpr std::uint32_t CTP_NINPUTS = 46;    /// Max number of CTP inputs for all levels
 static constexpr std::uint32_t CTP_NCLASSES = 64;   /// Number of classes in hardware
 static constexpr std::uint32_t CTP_MAXL0PERDET = 5; /// Max number of LM/L0inputs per detector
-/// Positions of CTP Detector inputs in CTPInputMask
+/// Positions of CTP Detector inputs in CTPInputMask: first=offset, second=mask
+/// For digits input position is  fixed
+/// CTP hits are inputs. Digits are just inputs collected from all detectors.
+/// digits->raw: CTP Config to be used
+/// raw->digits: CTP config to be used
 static constexpr std::pair<uint32_t, std::bitset<CTP_MAXL0PERDET>> CTP_INPUTMASK_FV0(0, 0x1f);
 static constexpr std::pair<uint32_t, std::bitset<CTP_MAXL0PERDET>> CTP_INPUTMASK_FT0(5, 0x1f);
 ///
 struct CTPDigit {
   o2::InteractionRecord intRecord;
   std::bitset<CTP_NINPUTS> CTPInputMask;
-  std::bitset<CTP_NCLASSES> CTPClassMask;
```

Comment:
for the record: the ROOT schema evolution does not support change of N in the `bitset<N>`.
Are these `CTP_NINPUTS` and `CTP_NCLASSES` constants which never change?

Reply:
No, unless we change hardware.